cs/strings.texy

```diff
@@ -248,9 +221,52 @@ echo Strings::random(5, 'A-Z'); // 'HLKUR'
 \--
 
 
+Kódování
+========
+
+fixEncoding($s, $encoding=`'UTF-8'`) .{toc: fixEncoding()}
+------------------------------------------------------
+
+Odstraní z řetězce neplatné znaky v daném kódování.
+
+/--php
+$correctStrings = Strings::fixEncoding($string);
+\--
+
+
+*bool* checkEncoding($s, $encoding=`'UTF-8'`) .{toc: checkEncoding()}
+-----------------------------------------------------------------
+
+Zjistí, jestli jde o řetězec platný v daném kódování.
+
+/--php
+$isUtf8 = Strings::checkEncoding($string);
+\--
+
+
+toAscii($s)
+-----------
+
+Odstraní z UTF-8 řetězce diakritiku.
```

Comment:
Tohle asi není úplně správně. Odstraní to navíc všechny non-ascii znaky, tj. i věci jako unicode smajlíky apod.